src/styles.css

```diff
@@ -468,7 +490,7 @@ nav .burger-icon.burger-open {
 }
 .temp-text {
   fill: #efefef;
-  font-size: 0.75rem;
+  font-size: 14pt;
```

Comment:
why not variable? why not rem units?

Reply:
Because this is for temperature SVG and it is being scaled on its own.